examples/bank_account/bank_account.py

```diff
@@ -16,7 +16,7 @@ def info(self):
     def deposit(self, amount):
         """ deposit money """
         if amount > 0:
-            self._balance += amount - self._calc_commission_rate(self._hasCommissionDiscount)
+            self._balance -= amount - self._calc_commission_rate(self._hasCommissionDiscount)
```

Comment:
**Suggestion:** The balance update operation in the deposit method seems to be incorrect. The balance should increase when a deposit is made, not decrease.
```suggestion
    def deposit(self, amount):
        """ deposit money """
        if amount > 0:
            self._balance += amount - self._calc_commission_rate(self._hasCommissionDiscount)
        else:
            raise ValueError("deposit amount must be larger than 0")
```